lua/lspimport/init.lua

```diff
@@ -127,8 +113,9 @@ LspImport.import = function()
             vim.notify("no unresolved import error")
             return
         end
-        local diagnostic = get_diagnostic_under_cursor(diagnostics)
-        lsp_completion(diagnostic or diagnostics[1])
+        for _, diagnostic in ipairs(diagnostics) do
```

Comment:
Did you actually test this? The problem with this approach is that completion API is asynchronous i.e. the handlers are are executed as callbacks; thus we gonna chain a list of completions into _promises_. Another issue is that diagnostic line changes, so we gotta account for that after resolving an import in the list.